service-base/src/main/java/org/oskari/util/Customization.java

```diff
@@ -0,0 +1,96 @@
+package org.oskari.util;
+
+import fi.nls.oskari.cache.Cache;
+import fi.nls.oskari.cache.CacheManager;
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+import fi.nls.oskari.util.IOHelper;
+import fi.nls.oskari.util.JSONHelper;
+import fi.nls.oskari.util.PropertyUtil;
+import org.json.JSONArray;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Customization {
+    private static final String SVG_MARKERS_JSON = "svg-markers.json";
+    private static Cache<String> CUSTOMIZATION = CacheManager.getCache("Customization");
```

Comment:
This is a bit of an overkill if we can't get the logo byte[] here as well. Maybe even if we could store the logos here.